<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseMessage</class>
 <widget class="QDialog" name="PauseMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="message8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FOR EDITING</string>
   </property>
  </widget>
  <widget class="QLabel" name="message1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>YOU HAVE JUST PAUSED.</string>
   </property>
  </widget>
  <widget class="QLabel" name="message6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PRESS &quot;MENU&quot; AND SELECT</string>
   </property>
  </widget>
  <widget class="QLabel" name="message5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
  </widget>
  <widget class="QLabel" name="message7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;EDIT LINE AND MARK&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="markDataLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="markLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>MARK:</string>
   </property>
  </widget>
  <widget class="QLabel" name="message2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>YOU CURRENTLY GOING TO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineDataLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>LINE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="message3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PRESS ANY KEY TO CLEAR THIS MESSAGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="message4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>THEN PRESS &quot;MARK&quot; TO START NEXT LINE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
